<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aowForm</class>
 <widget class="QWidget" name="aowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>窗口的艺术</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>窗口的艺术</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aow1Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口抖动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aow2Btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>47</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口飘动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aow3Btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>47</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>大小变换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>77</y>
     <width>255</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
